src/Android/Services/DeviceActionService.cs

```diff
@@ -964,5 +964,14 @@ public async Task SetScreenCaptureAllowedAsync()
             }
             activity.RunOnUiThread(() => activity.Window.AddFlags(WindowManagerFlags.Secure));
         }
+
+        public void OpenAppSettings()
+        {
+            var intent = new Intent(Android.Provider.Settings.ActionApplicationDetailsSettings);
+            intent.AddFlags(ActivityFlags.NewTask);
+            var uri = Android.Net.Uri.FromParts("package", Application.Context.PackageName, null);
+            intent.SetData(uri);
+            Application.Context.StartActivity(intent);
+        }
```

Comment:
Could you check if we can also use this method on `OpenAccessibilityOverlayPermissionSettings()` in the same file? it seems to be opening the app settings as well in there so maybe we could reuse this method there as well

Reply:
It's not the same thing, it is android specific and it's to add "on top" permissions.